utils/gyb_syntax_support/DeclNodes.py

```diff
@@ -662,31 +662,27 @@
                        'PrefixOperatorToken',
                        'PostfixOperatorToken',
                    ]),
-             Child('InfixOperatorGroup', kind='InfixOperatorGroup',
+             Child('OperatorPrecedenceAndTypes', kind='OperatorPrecedenceAndTypes',
                    description='''
-                   Optionally specify a precedence group
+                   Optionally specify a precedence group and designated types.
                    ''',
                    is_optional=True),
          ]),
 
+    Node('IdentifierList', kind='SyntaxCollection',
```

Comment:
This probably needs another production that includes a trailing comma after each identifier.

Reply:
I patterned this on what was happening for inheritance with `InheritedTypeList`. I don't see any notion of a comma-separated list there. Am I missing something?